docs/user-guide/03-convenience-features.md

```diff
@@ -375,8 +375,11 @@ int test(MyType* pObj)
 
 Pointer types can also be specified using the generic syntax: `Ptr<MyType>` is equivalent to `MyType*`.
 
-> #### Note
-> Slang currently does not support pointers to immutable values, i.e. `const T*`.
+### Limitations
+
+- Slang supports pointers to global memory, but not shared or local memory.
```

Comment:
We also don't support producing `coherent` load/stores, or specifying custom alignment.

Reply:
We can add that "For example, it is invalid to take the pointer of a local variable."

Reply:
Just noticed this.  I added an extra bit explaining the bullet.